.gitignore

```diff
@@ -6,3 +6,4 @@ Gemfile.lock
 .rvmrc
 *.sw*
 *.orig
+.idea/
```

Comment:
I'd prefer to keep editor-specific ignore values out of the gitignore file.

My recommendation would be to add these to your global .gitignore for your local dev environment.